backend/compact-connect/lambdas/provider-data-v1/tests/function/test_data_model/test_client.py

```diff
@@ -121,9 +122,9 @@ def test_get_licenses_sorted_by_family_name(self):
         # Verify sorting by family name (without getting into how duplicate family names are sorted)
         all_items = [*first_items, *resp['items']]
         family_names = [item['familyName'] for item in all_items]
-        self.assertListEqual(sorted(family_names), family_names)
+        self.assertListEqual(sorted(family_names, key=quote), family_names)
```

Comment:
Was this failing before without quotes? Or is this due to the additional tests on different languages?

Reply:
Adding the additional character sets via Faker is what required that I add `quote` to match the way they are sorted in the DB.